torch/csrc/jit/symbolic_script.cpp

```
@@ -375,6 +617,108 @@ const std::vector<std::string> functions = {
+
+            return torch.softmax(self, dim, dtype), backward
+
+        def ___interpolate_backward(grad,
+                                    input,
+                                    mode: str,
+                                    align_corners: bool):
+            output_size = grad.size()[2:]
+            input_size = input.size()
+            input_dim = len(input_size)
+            if input_dim == 3 and mode == 'nearest':
+                grad_input = torch.upsample_nearest1d_backward(grad, output_size, input_size)
+            elif input_dim == 4 and mode == 'nearest':
+                grad_input = torch.upsample_nearest2d_backward(grad, output_size, input_size)
+            elif input_dim == 5 and mode == 'nearest':
+                grad_input = torch.upsample_nearest3d_backward(grad, output_size, input_size)
+            elif input_dim == 3 and mode == 'linear':
+                grad_input = torch.upsample_linear1d_backward(grad, output_size, input_size, align_corners)
+            elif input_dim == 4 and mode == 'bilinear':
+                grad_input = torch.upsample_bilinear2d_backward(grad, output_size, input_size, align_corners)
+            elif input_dim == 5 and mode == 'trilinear':
+                grad_input = torch.upsample_trilinear3d_backward(grad, output_size, input_size, align_corners)
+            elif input_dim == 4 and mode == 'bicubic':
+                grad_input = torch.upsample_bicubic2d_backward(grad, output_size, input_size, align_corners)
+            elif input_dim == 3 and mode == 'area':
+                grad_input = ___adaptive_avg_pool1d_backward(grad, input, output_size)
+            elif input_dim == 4 and mode == 'area':
+                grad_input = ___adaptive_avg_pool2d_backward(grad, input, output_size)
+            elif input_dim == 5 and mode == 'area':
+                grad_input = torch.adaptive_avg_pool3d_backward(grad, input)
+            else:
+                # NEVER REACH HERE
```

might want to raise an exception